msmbuilder/libdistance/libdistance.pyx

```diff
@@ -126,6 +131,56 @@ def assign_nearest(X, Y, const char* metric, npy_intp[::1] X_indices=None):
         raise TypeError('X and y must be both float32 or float64')
 
 
+def cdist(XA, XB, const char* metric):
+    """cdist(XA, XB, metric):
+
+    Computes distance between each pair of the two collections of inputs.
+
+    Parameters
+    ----------
+    XA : array, shape = (n_samples, n_features) or md.Trajectory
+        The data array
+    XB : array, shape = (n_samples, n_features) or md.Trajectory
+        The reference array
+    metric : {"euclidean", "sqeuclidean", "cityblock", "chebyshev", "canberra",
+              "braycurtis", "hamming", "jaccard", "cityblock", "rmsd"}
+        The distance metric to use. metric = "rmsd" requires that X be of type
+        md.Trajectory; other distance metrics require that it be a numpy array
+
+    Returns
+    -------
+    Y : ndarray
+        A :math:`n_A` by :math:`n_B` distance matrix is returned.
+        For each :math:`i` and :math:`j`, the metric
+        ``dist(u=XA[i], v=XB[j])`` is computed and stored in the
+        :math:`ij` th entry.
```

Comment:
I think this docstring on the return value hasn't need updated for the return value being two-dimensional